pkg/subctl/operator/submarinerop/serviceaccount/ensure.go

```diff
@@ -73,47 +76,76 @@ func Ensure(restConfig *rest.Config, namespace string) (bool, error) {
 func ensureServiceAccount(clientSet *clientset.Clientset, namespace string) (bool, error) {
 	sa := &v1.ServiceAccount{ObjectMeta: v1meta.ObjectMeta{Name: OperatorServiceAccount}}
 	_, err := clientSet.CoreV1().ServiceAccounts(namespace).Create(sa)
+	if errors.IsAlreadyExists(err) {
```

Comment:
Good to move this into pkg/lighthouse/sericeaccounts/ensure.go and call from there, separation of code and all that.

Reply:
Done.